### ISSUE TITLE
Switch from pint to openff-units?

### ISSUE THREAD

**Issue by IAlibay** [openforcefield/protein-ligand-benchmark#30]
With several openfe (see guefe) and potentially future F@H alchemy components using `openff-units`, would it make sense to switch from `pint` to `openff-units`?

**Comment by dotsdl** [comment]
One point raised by @j-wags: angstroms get printed out as "A with a circle on top"; need to make sure serialization gets something like "angstrom" written out.

**Comment by mattwthompson** [comment]
Sorry I didn't have this repo watched previously - this can be worked around in one go:

```python
In [1]: from openff.units import unit

In [2]: val = unit.Quantity(1.2, unit.angstrom)

In [3]: str(val)
Out[3]: '1.2 Å'

In [4]: unit.default_format
Out[4]: '~'

In [5]: unit.default_format = ""  # This is the line that should be added in an application to switch everything over

In [6]: str(val)
Out[6]: '1.2 angstrom'
```

Setting the default string formatting to so-called ["short default"](https://pint.readthedocs.io/en/stable/formatting.html#string-formatting) is an [override](https://github.com/openforcefield/openff-units/blob/0.1.5/openff/units/units.py#L71) that takes place inside of `openff-units`; this has caused enough grief elsewhere that I might actually revert that in an upcoming release. But in any case it can be set here with that one-liner after importing the module. That should change string formatting to something closer to sane expectations, but if it doesn't we can create a custom formatting registry with custom behavior.

If there are other quirks you run into I would be happy to be looped in - whether or not use you use the package, any feedback is useful to me.

**Comment by mattwthompson** [comment]
FYI: I've released version 0.1.6 that un-does the `DEFAULT_UNIT_REGISTRY.default_format = "~"` I mentioned above.

```
$ python
>>> from openff.units import unit  # 0.1.5
>>> str(unit.Quantity(4, unit.angstrom))
'4 Å'
$ pip install git+https://github.com/openforcefield/openff-units.git@0.1.6
<SNIP>
Successfully installed openff-units-0.1.6
$ python
>>> from openff.units import unit
>>> str(unit.Quantity(4, unit.angstrom))
'4 angstrom'
```

**Comment by mattwthompson** [comment]
FYI: I've released version 0.1.6 that un-does the `DEFAULT_UNIT_REGISTRY.default_format = "~"` I mentioned above.

```
$ python
>>> from openff.units import unit  # 0.1.5
>>> str(unit.Quantity(4, unit.angstrom))
'4 Å'
$ pip install git+https://github.com/openforcefield/openff-units.git@0.1.6
<SNIP>
Successfully installed openff-units-0.1.6
$ python
>>> from openff.units import unit
>>> str(unit.Quantity(4, unit.angstrom))
'4 angstrom'
```
